nodl_python/nodl/index/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Canonical Ltd.
```

Comment:
There's no need to make this a package at this point, let's just leave it as a module. I don't think making that module public makes much sense either: the name `index` leaks implementations details and has no meaning unless you understand that these functions relate to the ament index, which are details only we know. Unless we can think of a more suitable name, let's make it a non-public module and import the method we want to be public into `nodl/__init__.py`.